web/packages/teleterm/electron-builder-config.js

```diff
@@ -142,6 +142,29 @@ module.exports = {
   },
   win: {
     target: ['nsis'],
+    sign: customSign => {
+      if (process.env.CI != "true") {
```

Comment:
The variable will be a string rather than a bool. It can also be set to other values (such as "false") in some cases. Is this implicit type conversion a safe approach?

Reply:
Okay, I thought it's a boolean. 
We may only want to replace `!=` with `!==` (strict equality check) to make sure that no type conversions happen.

Reply:
Yes, I have a Windows VM, I will test it later today!

Reply:
I tested the build and unfortunately it is still not signed. In the build logs I see: 
```
Invoke-SignBinary : The term 'New-Guid' is not recognized as the name of a cmdlet, function, script file, or operable 
  program. Check the spelling of the name, or if a path was included, verify that the path is correct and try again.
  At line:1 char:119
  + ... /build.ps1; Invoke-SignBinary -UnsignedBinaryPath "C:\a\teleport.e\te ...
  +                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
      + CategoryInfo          : ObjectNotFound: (New-Guid:String) [Invoke-SignBinary], CommandNotFoundException
      + FullyQualifiedErrorId : CommandNotFoundException,Invoke-SignBinary
```
so I think there is a problem with running the PS script?